spec/presenters/taxon_presenter_spec.rb

```diff
@@ -2,15 +2,20 @@
 
 RSpec.describe TaxonPresenter do
   describe "#payload" do
-    it "generates a valid payload" do
-      presenter = TaxonPresenter.new(
+    let(:presenter) do
+      TaxonPresenter.new(
         title: "My Title",
         base_path: "/taxons/my-taxon"
       )
+    end
+    subject { presenter.payload }
```

Comment:
I think `subject` only introduces mental overhead when reading the tests.  

```
expect(subject).to be_valid_against_schema('taxon')
```

vs

```
expect(payload).to be_valid_against_schema('taxon')
```

WDYT?